packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -309,7 +309,7 @@ function ListView({
                     { target: headerLayoutTitle }
                   )}
                   placeholder={formatMessage({
-                    id: 'app.component.search.placeholder',
+                    id: 'content-manager.components.Search.placeholder',
```

Comment:
actually the `content-manager.components.Search.placeholder` translation returns "Search for an entry..." 🤔
after discussing it with the designer, it should be changed by only "Search" without the dots!
Would you want to update it or I could handle it in another PR if you prefer 😊

Reply:
The translation text has been updated.

Reply:
thank you! 🙏